Makefile

```diff
@@ -1,6 +1,6 @@
 gcs_cache_path := --gcs-cache-path=gs://zenodo-cache.catalyst.coop
 covargs := --append
-pytest_args := --durations 20 ${gcs_cache_path} --cov-fail-under=0
+pytest_args := --durations 20 ${gcs_cache_path} --cov-fail-under=0 -s
```

Comment:
non-blocking: `-s` has been around and a stable part of `pytest` forever... do you know _why_ the `-s` is now needed while the logs were working before? Some sort of dagster change?

Reply:
Making this `--capture=no` rather than the shorthand would make it a little clearer what is being done, in case someone has to debug this behavior in the future.